h2o-extensions/xgboost/src/main/java/hex/schemas/XGBoostV3.java

```diff
@@ -90,8 +90,6 @@
 
     @API(help="(same as n_estimators) Number of trees.", gridable = true)
     public int ntrees;
-    @API(help="(same as ntrees) Number of trees.", gridable = true)
-    public int n_estimators;
```

Comment:
I think we should either remove it or expose it to clients, @michalkurka WDYT?

Reply:
it is okay to remove from here if it was not exposed